examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -9164,8 +9164,8 @@ endpoint 1 {
     ram      attribute colorLoopStartEnhancedHue default = 0x2300;
     ram      attribute colorLoopStoredEnhancedHue default = 0x0000;
     ram      attribute colorCapabilities default = 0x1F;
-    ram      attribute colorTempPhysicalMinMireds default = 0x0000;
-    ram      attribute colorTempPhysicalMaxMireds default = 0xFEFF;
+    ram      attribute colorTempPhysicalMinMireds default = 0x009A;
```

Comment:
Do not merge (or resolve this comment) until spec PR and test plan PR are fully merged.